scripts/signature.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import sys
+
+import click
```

Comment:
AFAIK, most of the other scripts we have use argparse; any particular reason why don't use that, for consistency?

Reply:
I used click because I found it  easier to write decorators on functions and not have to deal with the routing logic etc, since I wanted to go with more git style multi command support. I can use argparse if you feel strongly enough.

Reply:
Made the change to use argparse.